reference/functions/hostswithgroup.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: hostswithgroup
+published: true
+---
+
+**This function is only available in CFEngine Enterprise.**
+
+[%CFEngine_function_prototype(group_name, field)%]
+
+**Description:** Returns a list from the CFEngine Database with the information
+`field` of all hosts sharing the group `group_name`.
+
+On CFEngine Enterprise, this function can be used to return a list of
+hostnames, ip-addresses or public key SHAs of hosts that are from the same group. It works very similarly as its twin `hostswithclass()`.
+
+[%CFEngine_function_attributes(group_name, field)%]
```

Comment:
Seems like we need mention of possible values for field.
As I understand it would be "name", "ip" or "hostkey"?